src/pages/[platform]/build-a-backend/data/data-modeling/relationships/index.mdx

```diff
@@ -24,14 +24,26 @@ export const getStaticPaths = async () => {
 export function getStaticProps(context) {
   return {
     props: {
-      
+
       meta
     }
   };
 }
 
 When modeling application data, you often need to establish relationships between different data models. In Amplify Data, you can create one-to-many, one-to-one, and many-to-many relationships in your Data schema. On the client-side, Amplify Data allows you to lazy or eager load of related data.
 
+<Callout warning>
+With versions of Amplify CLI `@aws-amplify/cli@12.12.2` and API Category `@aws-amplify/amplify-category-api@5.11.5`, an improvement was made to how relational field data is handled in subscriptions when different authorization rules apply to related models in a schema. The improvement redacts the values for the relational fields, displaying them as null or empty, to prevent unauthorized access to relational data.
```

Comment:
Why are we putting Gen 1 tooling in the Gen 2 docs (from the url it appears Gen 2)

Reply:
@sundersc Do we just need to remove the reference to Amplify CLI on Gen 2 pages? API Category still relevant to Gen 2, right?